lib/miq_helper.rb

```diff
@@ -0,0 +1,26 @@
+require 'pathname'
+require 'active_support/core_ext/object/blank'
+require 'active_support/string_inquirer'
+
+module Miq
```

Comment:
Since we already have a ManageIQ directory and namespace, can we reuse that one?